src/components/SavedCardDetails/SavedCardDetails.example.js

```diff
@@ -0,0 +1,53 @@
+import { fakeIntl } from '../../util/test-data';
+import SavedCardDetails from './SavedCardDetails';
+
+const noop = () => null;
+const defaultProps = {
+  intl: fakeIntl,
+  onDeleteCard: noop,
+  onChange: noop,
+  onManageDisableScrolling: noop,
+};
+
+export const SavedCardDetailsExample = {
+  component: SavedCardDetails,
+  props: {
+    ...defaultProps,
+    card: {
+      brand: 'visa',
+      expirationMonth: 10,
+      expirationYear: 2050,
+      last4Digits: '3220',
+    },
+  },
+  group: 'misc',
```

Comment:
You could also create new group if you want: e.g. `group: 'payment methods'` or something like that.